pkg/controller/resourcequota/resource_quota_controller.go

```diff
@@ -150,23 +172,21 @@ func NewResourceQuotaController(options *ResourceQuotaControllerOptions) (*Resou
 
 	if options.DiscoveryFunc != nil {
 		qm := &QuotaMonitor{
-			informersStarted:  options.InformersStarted,
-			informerFactory:   options.InformerFactory,
-			ignoredResources:  options.IgnoredResourcesFunc(),
-			resourceChanges:   workqueue.NewNamedRateLimitingQueue(workqueue.DefaultControllerRateLimiter(), "resource_quota_controller_resource_changes"),
-			resyncPeriod:      options.ReplenishmentResyncPeriod,
-			replenishmentFunc: rq.replenishQuota,
-			registry:          rq.registry,
+			informersStarted:      options.InformersStarted,
+			ignoredResources:      options.IgnoredResourcesFunc(),
```

Comment:
This becomes interesting. 

// TODO we either find a way to discover this or find a way to provide it via config

Reply:
Done

Reply:
Created https://github.com/kubernetes/kubernetes/pull/64310 to allow it to be provided via config (same as GC)